app/src/main/java/com/cornellappdev/volume/ui/screens/PublicationScreen.kt

```diff
@@ -0,0 +1,167 @@
+import com.cornellappdev.volume.analytics.VolumeEvent
+import com.cornellappdev.volume.data.models.Article
+import com.cornellappdev.volume.data.models.Publication
+import com.cornellappdev.volume.ui.components.general.CreateFollowPublicationRow
+import com.cornellappdev.volume.ui.components.general.CreateHorizontalPublicationRow
+import com.cornellappdev.volume.ui.components.onboarding.isScrolledToTheEnd
+import com.cornellappdev.volume.ui.states.PublicationsRetrievalState
+import com.cornellappdev.volume.ui.theme.VolumeOffWhite
+import com.cornellappdev.volume.ui.theme.VolumeOrange
+import com.cornellappdev.volume.ui.theme.notoserif
+import com.cornellappdev.volume.ui.viewmodels.PublicationsViewModel
+
+@Composable
+fun PublicationsScreen(
+    publicationsViewModel: PublicationsViewModel = hiltViewModel(),
+    //followButtonClicked: (Publication, Boolean) -> Unit,
+    onPublicationClick: (Publication) -> Unit
+) {
+    val publicationsUiState = publicationsViewModel.publicationsUiState
+    val lazyListState = rememberLazyListState()
+    val buttonClicked = rememberSaveable { mutableStateOf(false) }
+
+       Scaffold (topBar = {
+           // TODO fix positioning, little weird on my phone not sure if that's the case universally
+           Text(
+               modifier = Modifier.padding(start = 20.dp, top = 20.dp),
+               text = "Publications",
+               fontFamily = notoserif,
+               fontWeight = FontWeight.Medium,
+               fontSize = 28.sp,
+               textAlign = TextAlign.Left
+           )
+       },content = { innerPadding ->
+           LazyColumn(
+               modifier =
+               Modifier
+                   .fillMaxSize()
+                   .padding(start = 12.dp, top = innerPadding.calculateTopPadding()),
+           ) {
+               item {
+                   Text(
+                       modifier = Modifier.padding(top = 30.dp),
+                       text = "Following",
+                       fontFamily = notoserif,
+                       fontSize = 20.sp,
+                       fontWeight = FontWeight.Medium
+                   )
+
+                   Spacer(modifier = Modifier.height(10.dp))
+               }
+               item {
+                   when (val followingPublicationsState = publicationsUiState.followedPublicationsState) {
+                       PublicationsRetrievalState.Loading -> {
+                           Column(
+                               modifier = Modifier.fillMaxWidth(),
+                               horizontalAlignment = Alignment.CenterHorizontally
+                           ) {
+                               CircularProgressIndicator(color = VolumeOrange)
+                           }
+                       }
+                       PublicationsRetrievalState.Error -> {
+                           // TODO Prompt to try again, queryFollowingPublications manually (it's public). Could be that internet is down.
```

Comment:
Does this TODO still apply here as well? It says queryFollowingPublications here